src/components/my-journey/ContestCardTimer.tsx

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I think it would be better to extract the helper function `getRemainingTime `from the component and place it in a suitable folder (may be in `utils `folder).